qt/applications/workbench/workbench/widgets/about/presenter.py

```diff
@@ -26,32 +26,32 @@ class AboutPresenter(object):
     def __init__(self, parent, view=None, usage_reporting_verification_view = None):
         self.view = view if view else AboutView(parent, self, version_str(), release_date().strip())
         self.usage_reporting_verification_view = usage_reporting_verification_view \
-            if usage_reporting_verification_view  else UsageReportingVerificationView(parent, self)
+            if usage_reporting_verification_view else UsageReportingVerificationView(parent, self)
         self.parent = parent
-        self.view.clb_release_notes.clicked.connect(self.action_open_release_notes)
-        self.view.clb_sample_datasets.clicked.connect(self.action_open_download_website)
-        self.view.clb_mantid_introduction.clicked.connect(self.action_open_mantid_introduction)
-        self.view.clb_python_introduction.clicked.connect(self.action_open_python_introduction)
-        self.view.clb_python_in_mantid.clicked.connect(self.action_open_python_in_mantid)
-        self.view.clb_extending_mantid.clicked.connect(self.action_open_extending_mantid)
-        self.view.lbl_privacy_policy.linkActivated.connect(self.action_open_external_link)
-        self.view.pb_manage_user_directories.clicked.connect(self.action_manage_user_directories)
-        self.view.pb_close.clicked.connect(self.action_close)
+        self.view.about_widget.clb_release_notes.clicked.connect(self.action_open_release_notes)
```

Comment:
Could we just use a reference so if the widget ever changes either the name or the view someone doesn't have to go line by line in this function.
It's also significantly more readable from less boilerplate
```python
about_widget = self.view.about_widget
about_wdiget.clb_sample....
about_widget.clb_mantid_introduction...
about_widget.clb_python_introduction
# etc
```
The others have <3/4 references so it would be overkill to make this change in those

Reply:
Yes, sensible. Done.